### PR TITLE
Move CaprureResponse generation out of LinuxTracingGrpcHander

### PR DESCRIPTION
**Pull request by dimitry-**
This commit separates sending over gprc logic from
CaptureEvent generation part. LinuxTracingGrpcHander
no longer depends on GRPC part it is only resposible for
generating and sending CaptureEvents.

Test: Start Orbit, start capture - see that it works.

### CODE REVIEW

File: OrbitService/CaptureServiceImpl.cpp [hunk 1]
```diff
@@ -12,11 +13,51 @@ namespace orbit_service {
...
+
+using orbit_grpc_protos::CaptureEvent;
+
+class CaptureResponseListenerImpl final : public CaptureResponseListener {
+ public:
+  explicit CaptureResponseListenerImpl(
+      grpc::ServerReaderWriter<orbit_grpc_protos::CaptureResponse,
+                               orbit_grpc_protos::CaptureRequest>* reader_writer)
+      : reader_writer_{reader_writer} {}
+  void ProcessEvents(std::vector<orbit_grpc_protos::CaptureEvent>&& events) override {
```
Comment (dpallotti) [inline OrbitService/CaptureServiceImpl.cpp:26]:
```suggestion
      : reader_writer_{reader_writer} {}

  void ProcessEvents(std::vector<orbit_grpc_protos::CaptureEvent>&& events) override {
```

File: OrbitService/LinuxTracingGrpcHandler.h [hunk 2]
```diff
@@ -5,23 +5,21 @@
...
+#include "OrbitLinuxTracing/Tracer.h"
+#include "OrbitLinuxTracing/TracerListener.h"
 #include "absl/container/flat_hash_set.h"
 #include "absl/synchronization/mutex.h"
-#include "services.grpc.pb.h"
+#include "capture.pb.h"
 
 namespace orbit_service {
 
 class LinuxTracingGrpcHandler : public LinuxTracing::TracerListener {
```
Comment (dpallotti) [inline OrbitService/LinuxTracingGrpcHandler.h:19]: Should this just be renamed (back) to `LinuxTracingHandler` then, as it's independent from gRPC now?

File: OrbitService/CaptureServiceImpl.cpp [hunk 3]
```diff
@@ -12,11 +13,51 @@ namespace orbit_service {
...
+    if (events.empty()) {
+      return;
+    }
+
+    ORBIT_SCOPE("GrpcHandler::SendBufferedEvents");
```
Comment (dpallotti) [inline OrbitService/CaptureServiceImpl.cpp:31]: The string here should now be changed.

File: OrbitService/CaptureServiceImpl.cpp [hunk 4]
```diff
@@ -12,11 +13,51 @@ namespace orbit_service {
 using orbit_grpc_protos::CaptureRequest;
 using orbit_grpc_protos::CaptureResponse;
 
+namespace {
+
+using orbit_grpc_protos::CaptureEvent;
+
+class CaptureResponseListenerImpl final : public CaptureResponseListener {
```
Comment (dpallotti) [inline OrbitService/CaptureServiceImpl.cpp:20]: Could this have a more descriptive name? Like `GrpcCaptureResponseListener` or `GrpcCaptureResponseSender` or something like that?

Comment (dimitry-) [reply OrbitService/CaptureServiceImpl.cpp:20]: yes - this is better, renamed.

File: OrbitService/CaptureServiceImpl.cpp (hunk 1 above)
Comment (dimitry-) [reply OrbitService/CaptureServiceImpl.cpp:26]: done.

File: OrbitService/CaptureServiceImpl.cpp (hunk 3 above)
Comment (dimitry-) [reply OrbitService/CaptureServiceImpl.cpp:31]: done.

File: OrbitService/LinuxTracingGrpcHandler.h (hunk 2 above)
Comment (dimitry-) [reply OrbitService/LinuxTracingGrpcHandler.h:19]: Yeah, I was thinking about this and then forgot - done.

**Review by dpallotti — APPROVED** [review]

File: OrbitService/LinuxTracingHandler.h [hunk 5]
```diff
@@ -5,29 +5,27 @@
 #ifndef ORBIT_SERVICE_LINUX_TRACING_GRPC_HANDLER_H_
 #define ORBIT_SERVICE_LINUX_TRACING_GRPC_HANDLER_H_
```
[the remark this answers is not in the document]
Comment (dimitry-) [reply OrbitService/LinuxTracingHandler.h:6]: sure - done.

**Pull request merged**